### PR TITLE
Add and publish test coverage reports

### PR DESCRIPTION
**Pull request by SergKam**
All tests generate test coverage reports. We now publishing them to "codecov.io" for analyzing (if som one needs it) but without annoying PR comments.

The test now uses a prebuilt version of UI.
The test was failing randomly, depending on the host speed. I found that is dev mode every new test dynamically rebuilds the whole UI. It is because of dev mode assumes hot module reloading if underlying code file changes.
So to speed up tests and make them more reliable we test built production version, so it builds once and then used in all tests. The runs twice as fast minimum.

### CODE REVIEW

**Comment by rcillo** [comment]
@SergKam how to find them now?

**Comment by rcillo** [comment]
👍

**Comment by rcillo** [comment]
👍

File: tests/end2end/createEtForm.spec.js [hunk 1]
```diff
@@ -74,14 +74,17 @@ describe('Create Event type form', function() {
 
         this.browser.login('#createtype')
         .selectByValue(strategyInput, 'hash')
+        .sleep()
```
Comment (rcillo) [inline tests/end2end/createEtForm.spec.js]: For how long does it sleep? Isn't it better to wait on a predictable condition instead of waiting on time?

**Comment by SergKam** [comment]
The reports are actually pretty full and useful. https://codecov.io/gh/zalando-nakadi/nakadi-ui/list/test-coverage/
I also fixed the tests and found that `waitFoVisible(some)` don't work usually, because Elm not just hiding elements but actually remove them from DOM, so we need to use `waitForExists(some)` instead.

**Comment by SergKam** [comment]
:+1:

**Comment by rcillo** [comment]
👍

**Pull request merged**